map/section-1.md

```diff
@@ -20,4 +20,13 @@ We invite and encourage you to add any company, event or initiative that helps o
 6. You can add as many additional tags as you wish, but make sure that the default tags are preserved.
 
 ## About the map
-Our map is part of [mapoftomorrow.org](http://mapoftomorrow.org). On their website you can find a lot of interesting green companies & initiatives that are not related to film production but may still be interesting to you! If you stumble upon anything that is relevant to our industry but is not shown on our page please add the ***#greenfilm*** tag and the appropriate category tags to make it appear!
\ No newline at end of file
+Our map is part of [mapoftomorrow.org](http://mapoftomorrow.org). The great thing about it: It is public, free and everything is open source. On their website you can find a lot of interesting green companies & initiatives that are not related to film production but may still be interesting to you! If you stumble upon anything that is relevant to our industry but is not shown on our page please add the ***#greenfilm*** tag and the appropriate category tags to make it appear!
+
+#### Sustainability & quality assurance
+The map uses a ranking system to avoid greenwashing. You can join other volunteers who are keeping track of the changes made to the entries. Entries can be rated in six categories. They are called „the positive factors“ and are the following:
```

Comment:
How does ranking avoid greenwashing? Things can still be on the map. I would personally rephrase it to something like:
> The map uses a ranking system to highlight particularly sustainable options. If you have already worked with a company on the map, we would appreciate it if you would rate it! ....

Reply:
Maybe it does not avoid it completely but it is one aspect of it. Do you have an idea of another term?